cwms_radar_api/src/main/java/cwms/radar/api/SpecifiedLevelController.java

```diff
@@ -27,7 +56,13 @@
 
 
 public class SpecifiedLevelController implements CrudHandler {
+    static final String OFFICE = "office";
+    static final String TEMPLATE_ID_MASK = "template-id-mask";
+    static final String SPECIFIED_LEVEL_ID = "specified-level-id";
+    static final String FAIL_IF_EXISTS = "fail-if-exists";
```

Comment:
I'm beginning to think we need a class of just query parameter constants. some of these are used everywhere.

NOTE: this is for the record. no change required.

Reply:
Yeah that would help to to keep them consistent.